.buildkite/build_worksheet.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH=$(pwd)
+PIP_PATH="$SCRIPTPATH/env/bin/pip"
+PYTHON_PATH="$SCRIPTPATH/env/bin/python"
+
+echo "Now creating virtualenv..."
+virtualenv -p python3 env
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Can't create virtualenv."
+    exit 1
+fi
+
+PIP_CMD="$PIP_PATH install gspread==3.1.0"
+echo "Running $PIP_CMD..."
+$PIP_CMD
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Can't install '$PIP_CMD'."
+    exit 1
+fi
+
+PIP_CMD="$PIP_PATH install --upgrade oauth2client"
+echo "Running $PIP_CMD..."
+$PIP_CMD
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Can't install '$PIP_CMD'."
+    exit 1
+fi
+
+PIP_CMD="$PIP_PATH install PyOpenSSL"
+echo "Running $PIP_CMD..."
+$PIP_CMD
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Can't install '$PIP_CMD'."
+    exit 1
+fi
+
+PYTHON_CMD="$PYTHON_PATH .buildkite/create_integration_testing_worksheet.py"
+$PYTHON_CMD
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Can't execute '$PYTHON_CMD'."
+    exit 1
+fi
+
+buildkite-agent artifact upload '.buildkite/spreadsheet-link.txt'
```

Comment:
Caveat: I'm not super familiar with buildkite so I don't know best practices.

That said, this file seems a little strange to me: I would not have expected a shell script to be creating a virtual environment, installing pip dependencies one at a time, and then spawning other pipeline processes.

Instead, I would have expected something like:

* the build would be isolated in a docker environment, so a python virtualenv would not be needed
* the dependencies for this pipeline step would be defined in a pip `requirements.txt` file or a Pipenv `Pipfile`
* the individual steps (install dependencies, run `create_integration_testing_worksheet.py`, upload artifacts) are specified as multiple commands in the 'Create integration testing' block of the `pipeline.yml` file

Reply:
@indirectlylit I created another [issue](https://github.com/learningequality/kolibri/issues/4953) for this one. I need time to read more about docker.